app/javascript/src/Users/utils/index.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import type { Feature } from 'Users/types'
+
+const FEATURE_NAMES: { [feature: Feature]: string } = {
+  portal: 'Developer Portal',
+  finance: 'Billing',
+  settings: 'Settings',
+  partners: 'Developer Accounts -- Applications',
+  monitoring: 'Analytics',
+  plans: 'Integration & Application Plans',
+  policy_registry: 'Policy Registry'
+}
+
+const FEATURES_GRANTING_SERVICE_ACCESS = ['partners', 'monitoring', 'plans']
+
+export function getFeatureName (feature: Feature): string {
+  if (feature in FEATURE_NAMES) {
+    return FEATURE_NAMES[feature]
+  }
+
+  throw new Error(`${feature} is not a known feature`)
+}
+
+export function isServicePermissionsGranted (features: Feature | 'services' | Array<Feature | 'services'>): boolean {
+  if (typeof features === 'string') {
```

Comment:
and here we could match for a specific type

Reply:
But it matches for a specific type: `string`, otherwise `array`.
Grouping Feature and 'service' in a type would end up being more convoluted:
```
isServicePermissionsGranted (features: FeatureOrServices | Array<FeatureOrServices>) {
  if (typeof features === FeatureOrServices) {
     ...
```
It reads almost the same and keep in mind `Feature` is just a collection of strings, not a type per se.